### PR TITLE
Show notification for when devtools build is slow

### PR DESCRIPTION
**Pull request by helin24**
Jul-29-2020 10-12-20

### CODE REVIEW

File: src/io/flutter/devtools/DevToolsManager.java [hunk 1]
```diff
@@ -317,6 +335,7 @@ public void onTextAvailable(@NotNull ProcessEvent event, @NotNull Key outputType
           catch (JsonSyntaxException e) {
             processHandler.destroyProcess();
           }
+          notification.expire();
```
Comment (jacob314) [inline src/io/flutter/devtools/DevToolsManager.java:338]: glad to see the notification is being closed as soon as devtools is open!

Comment (jacob314) [reply src/io/flutter/devtools/DevToolsManager.java:338]: what happens if notification.expire() is called before the notification is shown?
Will that call throw or will the subsequent call to Notification.Bus.notify show the notification?

**Comment by jacob314** [comment]
Discussed offline. 
lgtmlgtm

File: src/io/flutter/devtools/DevToolsManager.java (hunk 1 above)
Comment (helin24) [reply src/io/flutter/devtools/DevToolsManager.java:338]: I tested by putting an extra `notification.expire()` statement before calling `Notification.Bus.notify` and verified that no notification is shown in that situation. Also verified that external projects where this step takes negligible time does not display the notification.

**Pull request merged**